### DISCUSSION TITLE
FileUpload downloadable not working

### DISCUSSION THREAD

**Discussion by 000yuying** [filamentphp/filament#12599] [Help]
### Package

Panel builder

### Package Version

v3.1

### How can we help you?

When I attach ->downloadable() in FileUpload and click the download button, 
I only get the option to open the image, similar to ->openable(), 
instead of the expected download action. 

I try to save my file uploads on a private disk(gcs), but I am unable to download my files after they have been uploaded.

Has anyone else encountered the same issue as me?
Please help to provide a solution. 

Thank you.

![截圖 2024-05-03 17 53 17](https://github.com/filamentphp/filament/assets/109067769/8b958532-6404-46f5-9c5b-bbf97258458c)

**Comment by xalabama** [comment]
?
->previewable(false)
